pkg/clusteragent/admission/mutate/config.go

```diff
@@ -85,11 +85,11 @@ func InjectConfig(rawPod []byte, ns string, dc dynamic.Interface) ([]byte, error
 func injectConfig(pod *corev1.Pod, _ string, _ dynamic.Interface) error {
```

Comment:
There is a metric for an attempt and a metric for an error. Should there be a metric for success?

Reply:
(This comment applies for all the places with this pattern)

Reply:
I think we can know the number of successful mutations by checking the difference `MutationAttempts - MutationErrors`.

IMO this should be enough. WDYT?